cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -88,6 +90,36 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     });
   });
 
+  it('deactivating a kontainer driver should hide its card from the cluster creation page', () => {
```

Comment:
To ensure customer use case is covered we should either (ideally) have a test that entirely removes the the kontainer drivers or request manual QA for them. We could be sneaky with the former and just intercept the kontainer request and return an empty list.

Sorry should have thought of this yesterday

Reply:
That worked fine for me. I did push a change with it and some tweaks
- prefixed the intercept url with `/`. don't think it's needed, but good to have
- added a check to ensure the groups are correct (no cluster drivers === no top group). i didn't noticed a couple of weird things whilst doing so
  - the cypress screenshot / live browser acts incorrectly, sometimes this would show the hidden row. guessing this is somehow related to the rendering then not using the tests intercepted api request
  - when listing group titles `first` and `eq` behaved weirdly given the live browser